core/templates/dev/head/filters.js

```diff
@@ -336,4 +336,34 @@ oppia.filter('summarizeDefaultOutcome', ['$filter', function($filter) {
     }
     return summary;
   }
-}])
+}]);
+
+oppia.filter('appendMetrixPrefix', [function() {
+  return function (input) {
+    return Math.abs(Number(input)) >= 1.0e+9
```

Comment:
sure! changed that. I think number should be checked if is negative or not  before you apply the filter, What do you think? and  for this case of viewers count do we need to log any error since the numbers are positive(>0)?